src/ui/js/Newgame.js

```diff
@@ -48,15 +50,7 @@ Newgame.showLoggedInPage = function() {
   }
 
   // Get all needed information, then display newgame page
-  Newgame.getNewgameData(Newgame.showPage);
-};
-
-Newgame.getNewgameData = function(callback) {
-  Env.callAsyncInParallel(
-    [
-      { 'func': Api.getButtonData, 'args': [ null ] },
-      Api.getPlayerData,
-    ], callback);
+  ButtonSelection.getNewgameData(Newgame.showPage);
```

Comment:
Hmm.  Maybe rename this function to `getButtonSelectionData()` - i agree that it's probably fine to move it, but the name becomes confusing when it's moved.